packages/data-stores/README.md

```diff
@@ -42,7 +39,9 @@ When an application needs to use a store that should be configured, it may be he
 ```ts
 // vertical-store.ts
 import { DomainSuggestions } from '@automattic/data-stores';
```

Comment:
Seems like we can remove this import completely as it's not used at all.

Reply:
Fixed, thanks!